packages/pf-codemods/README.md

```diff
@@ -514,6 +514,38 @@ Out:
 <DataList      />
 ```
 
+### datalist-remove-selectableRow [(#8827)](https://github.com/patternfly/patternfly-react/pull/8827)
+
+We've removed the selectableRow.onChange property and replaced it with onSelectableRowChange. The order of the params in the callback has also been updated so that the event param is first. This will require
+
+#### Example of manual change needed
```

Comment:
🔥 I like the idea of this. This one *might* be able to have a fix applied as a followup, but for right now this is a good fallback and might be good to look at our other codemods that don't provide a fix.